### PR TITLE
pv-scripts: added addtional config override hooks

### PR DESCRIPTION
**Pull request by friewerts**
* webpack.config.module.js
* webpack.config.legacy.js
* webpack.config.dev.module.js
* webpack.config.prod.module.js
* webpack.config.dev.legacy.js
* webpack.config.prod.legacy.js

### CODE REVIEW

File: packages/pv-scripts/README.md [hunk 1]
```diff
@@ -73,12 +73,30 @@ For further customization of the webpack-config, specific config-files can be ad
 **webpack.config.js:**
 Valid webpack.config files which will be merged with both (dev/prod) default configs.
```
Comment (deveth0) [inline packages/pv-scripts/README.md]: shouldnt it be singular: "Valid webpack.config file which will be merged.."?

File: packages/pv-scripts/helpers/prepareWebpackConfig.js [hunk 2]
```diff
@@ -2,11 +2,34 @@ const webpackMerge = require("webpack-merge");
 const { getConfig, getCustomWebpackConfig } = require("@pro-vision/webpack-config");
 
 async function prepareWebpackConfig(mode) {
-  const customModeConfigName = mode === "development" ? "webpack.config.dev.js" : "webpack.config.prod.js";
+  const customModeConfigName = mode === "development" ? "webpack.config.dev" : "webpack.config.prod";
   const customWebpackConfig = await getCustomWebpackConfig("webpack.config.js");
-  const customWebpackDevConfig = await getCustomWebpackConfig(customModeConfigName);
+  const customWebpackModuleConfig = await getCustomWebpackConfig("webpack.config.module.js");
+  const customWebpackLegacyConfig = await getCustomWebpackConfig("webpack.config.legacy.js");
+  const customWebpackStageConfig = await getCustomWebpackConfig(`${customModeConfigName}.js`);
```
Comment (deveth0) [inline packages/pv-scripts/helpers/prepareWebpackConfig.js]: wouldn't customWebpackModeConfig be more consistent? above it's also called customModeConfigName and not customStageConfigName

File: packages/pv-scripts/README.md (hunk 1 above)
Comment (friewerts) [reply packages/pv-scripts/README.md]: fixed

File: packages/pv-scripts/helpers/prepareWebpackConfig.js (hunk 2 above)
Comment (friewerts) [reply packages/pv-scripts/helpers/prepareWebpackConfig.js]: fixed

**Pull request merged**